docs/existing-vpc.rst

```diff
@@ -64,7 +64,7 @@ For example, if we have the following infrastructure (notation in terraform)::
 		resource "aws_nat_gateway" "main" {
 		  depends_on    = ["aws_internet_gateway.main"]
 		  allocation_id = "${aws_eip.nat.id}"
-		  subnet_id     = "${aws_subnet.public.1.id}"
```

Comment:
Not too sure but I guess this will break for length(AZ) > 1. I think 0 would be better or create a NAT per public subnet.

Thoughts @kragniz

Reply:
+1 I'd probably go for a seperate nat for each gateway.

Something like
```terraform
resource "aws_nat_gateway" "main" {
  count         = "${length(aws_subnet.public)}"
  depends_on    = ["aws_internet_gateway.main"]
  allocation_id = "${aws_eip.nat.id}"
  subnet_id     = "${aws_subnet.public.*.id[count.index]}"
}
```